docs/docs/concepts/flows.mdx

```diff
@@ -483,6 +483,44 @@ flow is completed.
   scope: "flow"
 ```
 
+#### Slot value validation
+
+You can now write slot value validation directly in your flow yaml file without the need to use custom actions.
+Add a `rejections` property to your `collect_information` step to define the validation rules.
+The `rejections` section is a list of mappings, where each mapping has an `if` and an `utter` property:
+- the `if` property is a condition written in natural language that is evaluated using the
+[pypred](https://github.com/armon/pypred) library.
+- the `utter` property is the name of the response the assistant will send if the condition evaluates to `True`.
+
+Here is an example:
+
+```yaml
+flows:
+  verify_eligibility:
+    description: This flow verifies if the user is eligible for creating an account.
+    name: verify eligibility
+    steps:
+      - id: "ask_age"
+        collect: "age"
+        rejections:
+          - if: age < 1
+            utter: utter_invalid_age
+          - if: age < 13
+            utter: utter_age_at_least_13
+        next: "ask_email"
+```
+
+Upon every validation failure, the assistant will again make another attempt to collect the information.
+This will be repeated until none of the rejection conditions pass. Once all rejections have evaluated to `False`, the
```

Comment:
I am not sure what you mean by `this` here. Can you explain what is repeated exactly?

Reply:
The bot re-asking the `utter_ask` response in order to collect the information. I'll make it more explicit.